src/content/docs/style-guide/processes-procedures/use-content-types-text-formats.mdx

```diff
@@ -71,9 +71,8 @@ Thr top of every doc begins with a set of metadata. Read on for more information
 
     <tr>
       <td>watermark</td>
-      <td>We use watermarks for things like beta content or to show that something is an internal New Relic doc. Commonly used watermarks are: BETA, Legacy, Deprecated, NR ONLY, etc.
-      
-      Watermarks look like large, faded text behind the doc's content.</td>
+      <td>Watermarks are customizible text backgrounds that indicate something special about a page. They look like large, faded text behind the doc's content. We used to insert watermarks in beta docs, but now we insert a [custom callout](/docs/style-guide/quick-reference/callouts#tr-custom-callout). Here are some other examples of watermarks: Legacy, Deprecated, NR ONLY, etc.      
```

Comment:
Looks good! The link goes where you want and examples how custom callouts are added! I'm curious if the "etc." is necessary here. How do you feel about rewriting that last sentence to say, "Other examples of watermarks include Legacy, Deprecated, and NR ONLY".

Reply:
@kaileyhaynes, thanks for the tip!